src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -163,7 +164,7 @@ void OnFullScreenButtonClick(object sender, RoutedEventArgs e)
 				popup.Child = null;
 				fullScreenGrid.Children.Clear();
 			}
-			fullScreenButton.Content = exitFullScreenIcon;
+			fullScreenButton.Content = fullScreenIcon;
```

Comment:
This was previously `exitFullScreenIcon`. Is changing it to `fullScreenIcon` a typo, or a bug fix?

Reply:
bug fix